.drone.star

```diff
@@ -19,7 +24,7 @@ config = {
     "phpunit": {
         "allDatabases": {
             "phpVersions": [
-                "7.3",
+                "7.4",
```

Comment:
@kiranparajuli589 you will need to remove the "reducedDatabases" section. We just want to test once with PHP  7.4 now.